biogeophys/FatesHydroWTFMod.F90

```diff
@@ -425,32 +427,42 @@ function th_from_psi_vg(this,psi) result(th)
     real(r8)             :: satfrac       ! Saturated fraction [-]
     real(r8)             :: th            ! Volumetric Water Cont [m3/m3]
 
+    real(r8)             :: psi_interp      ! psi where we start lin interp [Mpa]
+    real(r8)             :: th_interp       ! th where we start lin interp
     real(r8)             :: dpsidth_interp  ! change in psi during lin interp (slope)
-    real(r8)             :: m               ! pore size distribution param (1/n)
+    real(r8)             :: m               ! pore size distribution param 1
+    real(r8)             :: n               ! pore size distribution param 2    
 
-    m   = 1._r8/this%psd
+    m = this%m_vg
+    n = this%n_vg
 
-    if(psi>this%psi_max) then
+    ! pressure above which we use a linear function, 
+    ! psi_interp = -(1._r8/this%alpha)*(max_sf_interp**(1._r8/(-m)) - 1._r8 )**(1/n)
 
-        ! Linear range for extreme values
-        th = this%th_linear_sat(psi)
-        
-    elseif(psi<this%psi_min) then
-        
-        ! Linear range for extreme values
-        th = this%th_linear_res(psi)
+    ! Junyan modified to get rid of the linear interperation
+      psi_interp = 0
```

Comment:
why get rid of the linear interpolation?

Reply:
the linear interpolation code was left in, an issue was created, see #811